### ISSUE TITLE
Leveling tracker text box does not resize properly

### ISSUE THREAD

**Issue by kjroach** [Lailloken/Lailloken-UI#96]
Not sure if this is maybe related to my somewhat unusual 2400x1600 resolution, but the leveling tracker text box does not size properly when changing pages unless I open the settings page and change either the font size or opacity. That fixes it until the page changes again. This unfortunately makes it pretty much unusable for me right now.

![image](https://user-images.githubusercontent.com/64495588/185727727-0b6e3bbe-1f39-4e9d-a255-c9979db3f9e7.png)

**Comment by Lailloken** [comment]
Could you try something for me? Open data\External Functions.ahk in a text-editor, go to the very end and add
`Gui 9: New, -DPIscale`

so that this block looks like this:

```
SetTextAndResize(controlHwnd, newText, fontOptions := "", fontName := "")
{
	Gui 9: New, -DPIscale
	Gui 9: Font, %fontOptions%, %fontName%
	Gui 9: Add, Text, R1, %newText%
	GuiControlGet T, 9: Pos, Static1
	Gui 9: Destroy
	GuiControl,, %controlHwnd%, %newText%
	GuiControl, Move, %controlHwnd%, % "h" TH " w" TW
}
```
Save it, then restart LLK-UI. Let me know if that fixes it.

**Comment by Lailloken** [comment]
Okay, I just tested it and that fixes it on my end. This will be included in the next update. I'm sorry you're the one to find this bug.

**Comment by kjroach** [comment]
No problem! It does appear to fix it for me as well, thanks and thanks for your great work on these tools!

**Comment by kjroach** [comment]
FYI, it looks like you mentioned this fix in the new release, but the new release is missing the DPIscale fix line.

**Comment by Lailloken** [comment]
Ugh, you're right. While preparing that release I noticed that a last-minute fix for 3.19 was lost because of too much branching in my local versions, and I had to put it back in. I then forgot to include the updated external functions because of that.
